contracts/Delay.sol

```diff
@@ -1,11 +1,102 @@
 pragma solidity ^0.4.24;
 
 import "@aragon/os/contracts/apps/AragonApp.sol";
+import "@aragon/os/contracts/common/IForwarder.sol";
 
-contract Delay is AragonApp {
+contract Delay is AragonApp, IForwarder {
 
-    function initialize() public onlyInit {
+    bytes32 public constant SET_DELAY_ROLE = keccak256("SET_DELAY_ROLE");
+    bytes32 public constant DELAY_EXECUTION_ROLE = keccak256("DELAY_EXECUTION_ROLE");
+
+    string private constant ERROR_CAN_NOT_EXECUTE = "DELAY_CAN_NOT_EXECUTE";
+    string private constant ERROR_CAN_NOT_FORWARD = "DELAY_CAN_NOT_FORWARD";
+
+    struct DelayedScript {
+        uint256 timeSubmitted;
```

Comment:
If we are adding the pause functionality maybe we could change this to `executionTime`  and update the logic around that, as we would have to update it's value when `resume()` is executed ?

Reply:
Forgot the do this myself so please do it when you work on this!